nuclia/sdk/kb.py

```diff
@@ -37,9 +37,13 @@ def __init__(self):
         self.logs = NucliaLogs()
 
     @kb
-    def list(self, **kwargs) -> ResourceList:
+    def list(
+        self, page: Optional[int] = None, size: Optional[int] = None, **kwargs
+    ) -> ResourceList:
         ndb: NucliaDBClient = kwargs["ndb"]
-        data: ResourceList = ndb.ndb.list_resources(kbid=ndb.kbid)
+        data: ResourceList = ndb.ndb.list_resources(
+            kbid=ndb.kbid, query_params={"page": page, "size": size}
```

Comment:
the query params should be strings.
Also, can you include them only if they are not None?